serialization/InformationModel/README.md

```diff
@@ -0,0 +1,307 @@
+when serialized at level 1 or 2
+3) if one element is nested within multiple other elements, multiple copies of the same data are carried in the payload 
+4) if elements are nested, they have different serialized values than when serialized individually
+5) if elements are nested, rules for property inheritance must be defined, resulting in even more serialized
+variants of the same element
+
+For these reasons, it is more scalable, understandable and efficient to serialize all elements in a payload
+separately, without nesting or alternate representations.
+
+### Logical Value Interface
+
+The logical value of an element is an *interface* - an answer to the question "what is the value of each
+property defined in the logical model". For convenience, logical values are displayed as data rather than
+a set of questions and answers, so when looking at examples the distinction between logical values and
+physical data must be kept in mind.
+A serializer produces a single-element payload from the logical value of an element; a de-serializer parses
+an element's logical value from the payload.
+For each supported data format, a serialization specification must define the serialized value of an element
+for each element type in the logical model:
+
+![Serialize Single Element](images/serialize-element.jpg)
+
+To serialize a group of logical elements, each element is serialized individually and the individual data
+items are combined into an "SPDX file". The easiest combining method is concatenation, but combining
+should also reduce the size of the combined value. Any generic lossless compression algorithm
+(e.g., zip, gzip, 7zip) will work, but SPDX defines a trivial element-specific combiner that:
+* performs namespace prefix substitution for element IDs
+* supplies default values for common creation-related properties
+* yields a combined file in the same format (JSON) as the individual elements
+
+![Serialize Multiple Elements](images/serialize-multiple.jpg)
+
+To deserialize an SPDX file, the combined value is decompressed then split into individual
+data items that are parsed into logical element values.
+
+### SPDX Serialization Examples
+
+The SPDX v3 model diagram includes some JSON examples, but their purpose is to illustrate and
+develop the logical model, not specify how to construct and validate a byte sequence.
+When serialization specifications are defined, examples are needed to illustrate and develop them.
+
+Figure 3 shows a JSON example from the diagram as of 12/19/2022:
+
+![Figure 3](images/diagram-sbom.jpg)
+
+This example contains three elements in a nested structure: an SBOM (null-sbom), a Package
+(spdx-tools-3.0.1), and a Person (iamwillbar). The first task is to determine the logical values of these
+three elements. 
+
+The logical values of these elements are:
+
+**Package:**
+```json
+{
+  "id": "urn:spdx.dev:spdx-tools-3.0.1",
+  "type": {
```

Comment:
In the SPDX 2.X JSON format, we took the approach of having a different list/array for each type rather than including a "type" field.  See https://github.com/spdx/spdx-spec/blob/844144b3785dbc6a35065eda3b9d36adda874540/examples/SPDXJSONExample-v2.3.spdx.json#L61 for an example.
I can see some advantages to having the "type" approach proposed, but since it introduces an incompatibility I would like to discuss before agreeing.  @armintaenzertng  - any thoughts on this as another SPDX tool contributor?

Reply:
I currently lean more towards having a `type`/`@type` property on each element as this will mark better compatibility to json-ld. I agree with David that "compatibility" with SPDX2 should mostly refer to lossless translation from v2 to v3, not retaining the same syntax (which will look drastically different with the new model, anyway).